src/views/app.jsx

```diff
@@ -31,6 +47,8 @@ export default class App extends ListenerComponent {
         }
         return (
             <div id='app'>
+                <button type='button' onClick={this.makeError}>Error!</button>
+                <ErrorModal active={ErrorStore.hasError()} onClose={this.closeError} message={this.state.errorMessage} />
```

Comment:
Probably better to use `this.state.hasError` instead of `ErrorStore.hasError()` here, and get it from the ErrorStore with the rest of the state.